spring-ai/postman/spring-ai.postman_collection.json

```diff
@@ -0,0 +1,56 @@
+{
+	"info": {
+		"_postman_id": "f4282fac-bfe5-45b9-aae6-5ea7c43528ee",
+		"name": "spring-ai",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "9576856"
+	},
+	"item": [
+		{
+			"name": "Generate poetry with genre and theme",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{baseUrl}}/ai/poetry?genre={{genre}}&theme={{theme}}",
+					"host": [
+						"{{baseUrl}}"
+					],
+					"path": [
+						"ai",
+						"poetry"
+					],
+					"query": [
+						{
+							"key": "genre",
+							"value": "{{genre}}"
+						},
+						{
+							"key": "theme",
+							"value": "{{theme}}"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Generate haiku about cats",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{baseUrl}}/ai/cathaiku",
+					"host": [
+						"{{baseUrl}}"
+					],
+					"path": [
+						"ai",
+						"cathaiku"
+					]
+				}
+			},
+			"response": []
+		}
+	]
+}
```

Comment:
minor: aren't we missing an empty line after? for lint verification purposes.. not sure though